README.md

```diff
@@ -1,2 +1,11 @@
 # Study-group
 Automation challenges through W&amp;C
+
+## How to run this program
+#### To run this program, use your terminal and type `node hello.js`, passing in either nothing, a first name or a first and last name. 
+
+#### To install cypress, use NPM. First, initialize an npm package with `npm init` using the terminal while navigating the topmost level of your project directory.
+#### Next, use the `npm install cypress --save-dev` command to install cypress locally to this directory.
```

Comment:
No need for the instructions on initializing package.json and installing cypress, since you have package.json on your branch, so for install just write "Run `npm install` ". And for the running just use `npx cypress open` 
(it's ok to create a separate script when necessary - but in this case it actually isn't since your script name is longer than actual script command :D )

Reply:
lol I just misunderstood the challenge prompt, makes perfect sense! Also I have always found it odd that the Cypress documentation specifies using cypress:open as a command to include in the script...so weird. Will ping when I edit the README.

Reply:
Changes made!